src/component/utils/exploration/__tests__/__snapshots__/wrapBlockNodes.js.snap

```diff
@@ -0,0 +1,799 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders block with child that have wrapperElement 1`] = `
+<div>
+  <ol
+    className="public__DraftStyleDefault__ol"
+    data-offset-key="A-0-0"
+  >
+    <blockquote
+      data-block={true}
+      data-editor="editor"
+      data-offset-key="A-0-0"
+    >
+      <div
+        className="public__DraftStyleDefault__block"
+        data-offset-key="A-0-0"
+      >
+        <ol
+          className="public__DraftStyleDefault__ol"
+          data-offset-key="B-0-0"
+        >
+          <li
+            data-block={true}
+            data-editor="editor"
+            data-offset-key="B-0-0"
+          >
+            <div
+              className="public__DraftStyleDefault__block"
+              data-offset-key="B-0-0"
+            >
+              <span
+                data-offset-key="B-0-0"
+                style={Object {}}
+              >
+                <span
+                  data-text="true"
+                >
+                  fist list item
+                </span>
+              </span>
+            </div>
+          </li>
+        </ol>
+      </div>
+    </blockquote>
+    <div
+      className="public__DraftStyleDefault__block"
+      data-offset-key="B-0-0"
+    >
+      <span
+        data-offset-key="B-0-0"
+        style={Object {}}
+      >
+        <span
+          data-text="true"
+        >
+          fist list item
```

Comment:
Something seems odd to me about this snapshot - do we expect to have two nested 'li' items, each with the text 'first list item'?

Also nit about misspelling of 'first' :)

Reply:
Yea - looking at this again, I still think I would expect this test to just output the first `blockquote`.

Reply:
@flarnie I think you mean two nested `ol` items, that's fixed now btw, about to upload the thing